examples/typescript/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as spsFrontend from "@tensorworks/lib-spsfrontend";
+
+// Apply default styling from Epic's frontend
+export const PixelStreamingApplicationStyles = new spsFrontend.PixelStreamingApplicationStyle();
+PixelStreamingApplicationStyles.applyStyleSheet();
+
+document.body.onload = function () {
+
+	// Create a config object.
+	// Note: This config is extremely important, SPS only supports the browser sending the offer.
+	const config = new spsFrontend.Config({ useUrlParams: true, initialSettings: { OfferToReceive: true, TimeoutIfIdle: true } });
+
+	// Create a Native DOM delegate instance that implements the Delegate interface class
```

Comment:
Comment is incorrect as DOM delegate and interface no longer exists

Reply:
this has been resolved in 0e6eff2d1024c7c46e395f432ff5c83d8e283aa4